com.unity.ml-agents/CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ removed when training with a player. The Editor still requires it to be clamped
 #### com.unity.ml-agents (C#)
 - Fix a compile warning about using an obsolete enum in `GrpcExtensions.cs`. (#4812)
 - CameraSensor now logs an error if the GraphicsDevice is null. (#4880)
+- Removed unnecessary memory allocations in `ActuatorManager.UpdateActionArray()` (#4877)
```

Comment:
@surfnerd I added our other PRs here. If this gets repetitive, we can group these into a nested list later.